resources/definitions/syndaveraxi2.def.json

```diff
@@ -0,0 +1,46 @@
+{
+    "version": 2, 
+    "name": "SyndaverAXI2",
+    "inherits": "fdmprinter",
+    "metadata": 
+{
+        "type": "machine",
+        "visible": true,
+        "author": "Syndaver3D",
+        "manufacturer": "Syndaver3D",
+        "file_formats": "text/x-gcode",
+        "supports_usb_connection": true,
+        "preferred_quality_type": "draft",
+        "machine_extruder_trains":
+        {
+            "0": "syndaveraxi2_extruder_0"
+        }
+    },
+
+    "overrides": {
+        "machine_name": { "default_value": "AXI2" },
+        "machine_shape": { "default_value": "rectangular"},
+        "machine_heated_bed": { "default_value": true },
+        "machine_width": { "default_value": 280 },
+        "machine_depth": { "default_value": 280 },
+        "machine_height": { "default_value": 280 },
+        "machine_center_is_zero": { "default_value": false },
+        "machine_head_with_fans_polygon": {
+            "default_value": [
+                [ 0, 0 ],
+                [ 0, 0 ],
+                [ 0, 0 ],
+                [ 0, 0 ]
+            ]
+        },
+        "gantry_height": { "value": "286" },
```

Comment:
Are you very sure about this? This should be the distance from the tip of nozzle to the gantry system. The pictures of the machine that i've found show that it's not that distance.

Reply:
@nallath Yes, it looks like you are correct. I will get that fixed and pushed y'alls way.  Thank you for the catch.